packages/components/table/src/table/style-helper.ts

```diff
@@ -43,8 +43,7 @@ function useStyle<T>(
   })
   const isGroup = ref(false)
   const scrollbarViewStyle = {
-    display: 'inline-flex',
-    flexDirection: 'column',
+    display: 'inline-block',
```

Comment:
Please do not change here, inline-block will make the wrong height of scroll-view

Reply:
This is to fix an unexpected gap when using `inline-flex` & `flex-direction: column`. If you have some better idea to fix the gap, then we can revert this change.

<img src="https://user-images.githubusercontent.com/26999792/158497992-f7f711ab-4be5-4bdd-ac0d-65a65fb773d5.png" width="400">

Reply:
> This is to fix an unexpected gap when using `inline-flex` & `flex-direction: column`. If you have some better idea to fix the gap, then we can revert this change.
> 
> <img alt="" width="400" src="https://user-images.githubusercontent.com/26999792/158497992-f7f711ab-4be5-4bdd-ac0d-65a65fb773d5.png">

It seems to have solved the problem 👍